src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp

```diff
@@ -109,18 +109,18 @@ static void regclass_graph_PreProcessSteps(py::module m) {
 
     steps.def(
         "convert_element_type",
-        [](ov::preprocess::PreProcessSteps& self, ov::element::Type type) {
+        [](ov::preprocess::PreProcessSteps& self, ov::element::Type type = {}) {
             return &self.convert_element_type(type);
         },
-        py::arg("type"),
+        py::arg("type") = ov::element::Type{},
         R"(
             Converts input tensor element type to specified type.
-            Input tensor must have openvino.Type.f32 data type.
+            Input tensor must have openvino.Type data type.
 
             Parameters
             ----------
             type : openvino.runtime.Type
-                Destination type.
+                Destination type. By default type will be taken from model input's element type.
 
             Returns
             ----------
```

Comment:
Please use newer syntax for docs if possible (apply to all cases below and above).

Example:
https://github.com/openvinotoolkit/openvino/blob/5f40ba9a236a3dbd202abd0feeead9588d0a4395/src/bindings/python/src/pyopenvino/core/compiled_model.cpp#L212-L223

CC @akuporos do you covering these cases as well?

Reply:
@nosovmik @jiwaszki yes, I'm covering it.

Reply:
I believe this should be done in separate PR, right?

Reply:
@nosovmik yes, Anastasia is currently working on that.